src/main/webapp/index.html

```diff
@@ -2,18 +2,15 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Twitter Bot</title>
```

Comment:
Create a new html page to test your changes, this index.html will cause merge conflicts with FE.